simperby/src/node/stf.rs

```diff
@@ -0,0 +1,288 @@
+//! The definition of the state transition function which deduces
+//! the next block from the current state, current block and the next given block.
```

Comment:
```suggestion
/// The definition of the state transition function which deduces
/// the next block from the current state, current block and the next given block.
```
Or is this a special format for comments that come at the very top of the source code?

Reply:
https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html#outer-and-inner-documentation